tooling/Microsoft.VisualStudio.RazorExtension/RazorInfo/RazorInfoToolWindow.cs

```diff
@@ -53,28 +58,69 @@ protected override void Dispose(bool disposing)
 
             if (disposing)
             {
-                _workspace.WorkspaceChanged -= Workspace_WorkspaceChanged;
+                _projectManager.Changed -= ProjectManager_Changed;
             }
         }
 
-        private void Reset(Solution solution)
```

Comment:
This code was based on the solution before, but now it's based on our project system.